lib/commands/generate.ts

```diff
@@ -0,0 +1,108 @@
+import { utimes } from "fs-extra";
+import * as path from "path";
+import { TemplateManager } from "../TemplateManager";
+import { Util } from "../Util";
+import { default as config } from "./config";
+
+const command = {
+	aliases: ["g"],
+	command: "generate",
+	desc: "Generates new project",
+	templateManager: null,
+	// tslint:disable-next-line:object-literal-sort-keys
+	builder: yargs => {
+		yargs
+			.command({
+				aliases: ["t"],
+				command: "template [name]",
+				desc: "Generates custom template",
+				// tslint:disable-next-line:object-literal-sort-keys
+				builder: {
+					"framework": {
+						alias: "f",
+						default: "jquery",
+						describe: "Framework name.",
+						type: "string"
+					},
+					"name": {
+						alias: "n",
+						default: "custom-template",
+						describe: "Template name.",
+						type: "string"
+					},
+					"skip-config": {
+						alias: "s",
+						default: false,
+						describe: "Runs generate command without updating the cli config.",
+						type: "boolean"
+					},
+					"type": {
+						alias: "t",
+						default: "js",
+						describe: "Framework type.",
+						type: "string"
+					}
+				},
+				handler: command.template
```

Comment:
Can you remove the `handler` config and just call the `template` in the **cli.ts** case manually with `await` since it's asynchronous. The yargs handler execution won't wait for the handler to finish.